src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+type Props = {
+  setFilter: (filter?: boolean) => void
+};
+
+enum TypeFilter {
+  'all',
+  'active',
+  'completed',
+}
```

Comment:
Create the right enum, please 
https://www.typescriptlang.org/docs/handbook/enums.html
```suggestion
enum FilterType {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
```

Reply:
i don't see need repead name, like this i use number and take string